backend/src/main/java/gov/cdc/usds/simplereport/idp/repository/LiveOktaRepository.java

```diff
@@ -185,11 +185,13 @@ public LiveOktaRepository(
           g.listUsers()
               .forEach(
                   u -> {
-                    String username = u.getProfile().getEmail();
-                    if (!usersToGroupNames.containsKey(username)) {
-                      usersToGroupNames.put(username, new HashSet<>());
+                    if (u.getStatus() != UserStatus.SUSPENDED) {
```

Comment:
felt like an important patch to sneak in

Reply:
👀

Reply:
Not a blocker: Wouldn't suspended users already not be able to log in?

Reply:
yes, but this is a method to retrieve other users in your organization. a user who has been deleted (SUSPENDED, in Okta) shouldn't come up in that user list